src/client/QXmppMovedManager.cpp

```diff
@@ -22,6 +22,12 @@
 using namespace QXmpp;
 using namespace QXmpp::Private;
 
+///
+/// \typedef QXmppMovedManager::Result
+///
+/// Empty result containing QXmpp::Success or a QXmppError
+///
+
```

Comment:
I suggest to move that below the documentation for `QXmppMovedManager`.

Reply:
Apart from that, how about simply using `QXmppClient::EmptyResult` which is already documented?